### DISCUSSION TITLE
Smaller magnifying glass icon in the search window?

### DISCUSSION THREAD

**Discussion by jjw24** [Flow-Launcher/Flow.Launcher#588] [Ideas]
Would you entertain the aesthetic of a smaller search icon?

**Before**
![image](https://user-images.githubusercontent.com/26427004/126572221-f92c4f9a-3b38-4ebc-b136-7a1d9a5ea3be.png)

**After**
![image](https://user-images.githubusercontent.com/26427004/126572061-892405ea-a776-4296-b31d-1684a29e3563.png)

A lot less in your face kinda feel. Thoughts?

@Flow-Launcher/team

**Comment by jjw24** [comment]
i am also open to a new prettier icon if anyone's got one or would like to make one :)

**Comment by deefrawley** [comment]
The current one probably could be reduced in size a bit. I don't mind a new icon but don't think it should be too detailed or distracting - simplicity is good here.
